client/src/hooks/useResponsive.tsx

```diff
@@ -0,0 +1,57 @@
+import { useTheme } from "@mui/material/styles";
+import useMediaQuery from "@mui/material/useMediaQuery";
+
+/**
+ * This hook is used to make the app responsive
+ * @param query - The query to be used
+ * @param start - The start breakpoint
+ * @param end - The end breakpoint
+ * @returns - The result of the query
+ * @description - This hook is used to make the app responsive
+ */
+
+export function useResponsive(query: string, start: any, end?: any) {
```

Comment:
We use the arrow function format for functions (() => {...}).


```suggestion
export const useResponsive = (query: string, start: any, end?: any) => {
```

Reply:
Why is the parameter `query` called "query"?
`start` and `end` are any typed. Can we make the type more explicit?